bypros/proxy_test.go

```diff
@@ -246,13 +246,18 @@ func TestProxyCatchUp_Wrong_Skipchain(t *testing.T) {
 		storage: storage,
 	}
 
-	_, stop, err := s.CatchUP(&CatchUpMsg{
+	out, stop, err := s.CatchUP(&CatchUpMsg{
 		ScID:        bct.Genesis.SkipChainID(),
 		Target:      bct.Roster.Get(0),
 		FromBlock:   bct.Genesis.SkipChainID(),
 		UpdateEvery: 9,
 	})
 
+	go func() {
```

Comment:
Can't you initialize the channel with a length of 1 so you can get rid of this go-routine?